src/harness/reference_models/move_list_calc/move_list.py

```diff
@@ -0,0 +1,392 @@
+
+    return grants_inside
+
+
+def computeInterference(cbsd_grant, constraint, num_iteration):
+
+    """
+    Calculate interference contribution of each grant in the neighborhood to
+    the protection constraint c.
+
+    Inputs:
+        cbsd_grant:     a dictionary containing installation and grant information of the CBSD
+        constraint: 	protection constraint, a tuple with named fields of
+                        'latitude', 'longitude', 'low_frequency', 'high_frequency'
+        num_iteration:  a number of Monte Carlo iterations
+
+    Returns:
+        interference: 	a dictionary containing CBSD index, median interference contribution,
+                        K random interference contributions of the grant to the protection
+                        constraint c, and bearing from c to CBSD location.
+    """
+
+    # Get CBSD grant information 	
+    grant_index = cbsd_grant.get('grantIndex')
+    lat_cbsd = cbsd_grant.get('latitude')
+    lon_cbsd = cbsd_grant.get('longitude')
+    h_cbsd = cbsd_grant.get('height')
+    indoor_cbsd = cbsd_grant.get('indoorDeployment')
+    ant_beamwidth_cbsd = cbsd_grant.get('antennaBeamwidth')
+    ant_az_cbsd = cbsd_grant.get('antennaAzimuth')
+    maxEirpPerMHz_cbsd = cbsd_grant.get('maxEirp')
+    low_freq_cbsd = cbsd_grant.get('lowFrequency')
+    high_freq_cbsd = cbsd_grant.get('highFrequency')
+
+    # Get protection constraint information (protection point and frequency range)
+    lat_c = constraint.latitude
+    lon_c = constraint.longitude
+    low_freq_c = constraint.low_frequency
+    high_freq_c = constraint.high_frequency
+    h_c = REF_INC_ANTENNA_HT
+
+    # Compute median and K random realizations of path loss/interference contribution
+    # based on ITM model as defined in [R2-SGN-03] (in dB)
+    reliabilities = np.random.rand(num_iteration)    # get K random reliability values from an
+                                                     # uniform distribution over [0,1)
+    reliabilities = np.append(reliabilities, [0.5])  # add 0.5 (for median loss) as
+                                            # a last value to reliabilities array
+    results = wf_itm.CalcItmPropagationLoss(lat_cbsd, lon_cbsd, h_cbsd, lat_c, lon_c, h_c,
+                                            indoor_cbsd,
+                                            reliability=reliabilities,
+                                            freq_mhz=FREQ_PROP_MODEL)
+    path_loss = np.array(results.db_loss)
+
+    # Compute CBSD EIRP inside the frequency range of protection constraint
+    eirpPerMHz_cbsd = antenna.GetStandardAntennaGains(results.incidence_angles.hor_cbsd,
+                                                      ant_az_cbsd, ant_beamwidth_cbsd,
+                                                      maxEirpPerMHz_cbsd)
+    eff_bandwidth = min(high_freq_cbsd, high_freq_c) - max(low_freq_cbsd, low_freq_c)
+    eirp_cbsd = eirpPerMHz_cbsd + 10 * np.log10(eff_bandwidth / 1000000.0)
```

Comment:
I am not sure we actually capture the EIRP per MHz here.

The constraint bandwidth is typically 10MHz (see your script for example). So eff_bandwidth is 10MHz. So we are taking eirpPerMHz_cbsd and adding 10dB. So we are effectively computing the EIRP per 10MHz here.
The threshold of -144dBm for DPA is to be compared to EIRP per Mhz, so in your script we have a 10dB off.
All this works only if the caller is passing a constraint freq range of exactly 1Mhz.

All the EIRP thing is a bit confusing... I think that at the end we need to compute a separate move list per SAS channel (ie 5MHz or 10MHz depending how latest discusionn goes in Winnforum), and that we simply need to check:
 1) do we intersect with the radar detected frequency (what you call the constraint frequency) - you do it properly in the findGrantsInsideNeighborhood() routine
 2) in that case, simply take the EIRP per MHz , since there is no fractional intersection of the radar bandwdidth

In conclusion the two linesshould be removed

Please think about this and let me know if I get this wrong !

Reply:
Update: I was wrong : the -144 is to be compared with the EIRP/10Mhz, I got confused by internal mismatched documentation. 
So I guess this change my previous remark, although I still think it is still valid in the principle of simply computing the power as eirpPerMhz +10dB, rather than eff_bandwidth ?? 
Please comment

Reply:
The constraint bandwidth is typically 10 MHz (e.g., 3600-3610 MHz in the move_list_example.py). However, the overlapping/intersecting bandwidth (denoted as 'eff_bandwidth') between the CBSD grant and the protection constraint frequency range <= 10 MHz. For example, if the CBSD grant has a frequency range of 3605-3610 MHz, eff_bandwidth = 5 MHz and 10*np.log10(eff_bandwidth / 1000000.0) = ~7 dB (not 10 dB). That's why, we need to compute 'eff_bandwidth'. By the way, 'eff_bandwidth' might not be a good name, I will change it to 'overlapping_bw' as in the findGrantsInsideNeighborhood(), what do you think? 

Yes, the protection threshold for co-channel offshore DPAs is -144 dBm/10MHz. More confusing is that the threshold for out-of-band inland DPAs is -125 dBm/MHz (not dBm/10MHz) (in R2-IPM-04). 

Please let me know if it's still not clear to you.